tests/_test_msui/test_mscolab.py

```diff
@@ -56,7 +56,7 @@ def setup(self, qtbot, mscolab_server):
         assert add_user_to_operation(path=self.operation_name, emailid=self.userdata[0])
         self.user = get_user(self.userdata[0])
 
-        self.main_window = msui.MSUIMainWindow(mscolab_data_dir=mscolab_settings.MSCOLAB_DATA_DIR)
+        self.main_window = msui.MSUIMainWindow(operations_data=mscolab_settings.OPERATIONS_DATA)
```

Comment:
Why does msui reuse a path from mscolab?

Reply:
using the same name is also not that good.

Reply:
The dir there is used when we switch "Work Asynchronously" on.